deploy/crds/infinispan.org_infinispans_crd.yaml

```diff
@@ -419,6 +419,28 @@ spec:
             dependencies:
               description: External dependencies needed by the Infinispan cluster
               properties:
+                artifacts:
+                  items:
+                    properties:
+                      hash:
+                        description: Hash sum of the downloaded file to validate
+                        pattern: ^(sha1|sha224|sha256|sha384|sha512|md5):[a-z0-9]+
+                        type: string
+                      type:
+                        description: Type of file (archive) that will be downloaded. If not specified, the file type will be automatically determined by its extension or downloaded without unpacking
```

Comment:
```suggestion
                        description: File type for the archive that you want to download. If not specified, the file type is automatically determined from the extension.
```

Reply:
@dmvolod Sorry but what does "or downloaded without unpacking" mean here? If you specify the file type then the archive is unpacked after download / if you don't specify the file type then it is not unpacked

Reply:
@oraNod it's possible to download files in thee ways: download any file type and place it into the folder, unpack archive with explicitly defined type (tar.gz o zip) and unpack archive based on the extension detection. In this case file is not archive only, it can be any file type customer wants to embedded to the server.

Reply:
@dmvolod Thanks! I updated the suggestion. That seems like a lot of info to fit into a single description without making things confusing. How about if we tackle the different ways to download files in docs?

Reply:
Thanks, @oraNod . Agree, it's better to explain all actions in doc. This description just for common information.